blogposts/2021/release-3.31.md

```diff
@@ -0,0 +1,88 @@
+  - description: Operator documentation has been added to the Search Reference sidebar section.
+    url: https://github.com/sourcegraph/sourcegraph/pull/23116
+    category: Search
+  - description: Syntax highlighting support for the Cue language.
+    url: https://cuelang.org
+    category: Search
+  - description: Reintroduced a revised version of the Search Types sidebar section.
+    url: https://github.com/sourcegraph/sourcegraph/pull/23170
+    category: Search
+  - description: "Added support for [`git p4`'s `--use-client-spec` option](https://git-scm.com/docs/git-p4#Documentation/git-p4.txt---use-client-spec) can now be enabled by configuring the `p4.client` field in the Perforce integration."
+    url: https://github.com/sourcegraph/sourcegraph/pull/23845
+    category: Admin
+  - description: "Batch Changes changesets can now be [set to published when previewing new or updated batch changes](https://docs.sourcegraph.com/batch_changes/how-tos/publishing_changesets#within-the-ui)."
+    url: https://github.com/sourcegraph/sourcegraph/issues/22912
+    category: Batch Changes
+  - description: "Code Insights drill-down filters now allow filtering insights data on the dashboard page using repo: filters."
+    url: https://github.com/sourcegraph/sourcegraph/issues/23186
+    category: Code Insights
+  - description: "Code Insights feature flag `DISABLE_CODE_INSIGHTS` environment variable has moved from the `repo-updater` service to the `worker` service. Any users of this flag will need to update their `worker` service configuration to continue using it."
+    url: https://github.com/sourcegraph/sourcegraph/pull/23050
+    category: Repositories
+  - description: "The Docker Compose Jaeger container's `SAMPLING_STRATEGIES_FILE` now has a default value. If you are currently using a custom sampling strategies configuration, you may need to make sure your configuration is not overridden by the change when upgrading."
+    url: "https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/489"
+    category: Admin
+  - description: The copy icon displayed next to files and repositories will now copy the file or repository path. Previously, this action copied the URL to clipboard.
+    url: https://github.com/sourcegraph/sourcegraph/pull/23390
+    category: Repositories
+  - description: "For Docker-Compose and Kubernetes users, the built-in main Postgres and codeintel databases have switched to an alpine Docker image. This requires re-indexing the entire database. This process can take up to a few hours on systems with large datasets."
+    url: https://github.com/sourcegraph/sourcegraph/pull/23697
+    category: Repositories
+  - description: "All commit search results for `and`-expressions are now highlighted."
+    url: https://github.com/sourcegraph/sourcegraph/pull/23336
+    category: Search
+  - description: Fixed a result streaming throttling issue that was causing significantly increased latency for some searches.
```

Comment:
Is this the same update:

 description: Results are now streamed from searcher by default, improving memory usage and latency for large, unindexed searches.

If they're related, this copy is a bit easier to understand.

Reply:
In this case, no, not related -- the one you quoted is about backend services talking to each other. I agree that the language can be made clearer though; I will adjust.